packages/composite-checkout/package.json

```
@@ -6,6 +6,14 @@
 	"module": "dist/esm/public-api.js",
 	"types": "dist/types/public-api.d.ts",
 	"calypso:src": "src/public-api.ts",
+	"exports": {
+		".": {
+			"calypso:src": "./src/public-api.ts",
+			"import": "./dist/esm/public-api.js",
+			"require": "./dist/cjs/public-api.js",
+			"types": "./dist/types/public-api.d.ts"
```

TypeScript wants "types" before "import"/"require".
```suggestion
			"types": "./dist/types/public-api.d.ts",
			"import": "./dist/esm/public-api.js",
			"require": "./dist/cjs/public-api.js"
```